### PR TITLE
Troubleshoot page + updated test checklist

### PR DESCRIPTION
**Pull request by jakrams**

### CODE REVIEW

File: Documentation/troubleshoot.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Once you are done using ARRT, you can reset the proxy with:
+
+```cmd
+netsh winhttp reset proxy
+```
+
+Or, if you already had another proxy before, use the set proxy command to revert the setting to that instead.
+
+## Connection to ARR sessions fail
```
Comment (FlorianBorn71) [inline Documentation/troubleshoot.md:42]: either "Connections" or "fails"

**Review by FlorianBorn71 — APPROVED** [review]

File: .pipelines/testchecklist.md [hunk 2]
```diff
@@ -3,207 +3,133 @@
...
-- Change some of the default configuration and start the conversion.
-- Open the Azure Storage Explorer and inspect the input blob container
-- Navigate to the input directory
-- Verify that a file "conversionSettings.json" is created besides the model file (also when the input root directory is set to a parent directory)
-- Verify that the file contains all of the non-default values that were set in ARRT
-  
-## Rendering panel
+- If it succeeds the icon and the status would reflect completed/failed status.
+- The time on the conversion should stop running and will indicate the total conversion time.
+- On success/failure a label below the advanced options should display the result ("finished successfully" or and error message)
```
Comment (MichaelZp0) [inline .pipelines/testchecklist.md:66]: nit - "or an error" typo

File: .pipelines/testchecklist.md [hunk 3]
```diff
@@ -3,207 +3,133 @@
...
+- Check that the *Max Time* and *Remaining Time* are the ones you specified
+- Check that now the *Stop* button is enabled
+- Click on *Extend Now* -> The specified time is added to both *Max Time* and *Remaining Time*
+- Wait a bit (or use a session with a shorter max time) and check the log to see that the auto-extension kicks in
+- Click the *Open ArrInspector* button and check that it's working
+
+### Models / Rendering / Camera
+
+- With a running session, test the *Load Model from Storage* and *Load Model with URL* buttons
+- There should be progress bar at the bottom when any model loading is ongoing
```
Comment (MichaelZp0) [inline .pipelines/testchecklist.md:93]: nit - There should be a progress bar

File: .pipelines/testchecklist.md [hunk 4]
```diff
@@ -3,207 +3,133 @@
...
+- Verify that the first loaded model gets centered in the viewport
+- You can add multiple models, use *Remove Models* to clear them all
+- Test that the *Model Scale* option works
+- Use the *Camera...* button to adjust camera options (near/far plane, camera speed)
+- In the settings, disconnect from the storage account (e.g. by the name to be invalid)
```
Comment (MichaelZp0) [inline .pipelines/testchecklist.md:98]: What do you mean with e.g. by the name to be invalid?
By setting the name to some invalid name, or doing  something else to make the current name invalid?

File: .pipelines/testchecklist.md [hunk 5]
```diff
@@ -3,207 +3,133 @@
...
-- Verify that loaded models is centered in the viewport
-- Try to change the global scale (remove the auto) and see if it works.
-- Apply auto global scale, load a model, then change the near and far plane to a very small range, and reload the model to see if the global scale (lowered) to adapt to the new range.
+- Open the *Camera* dialog, there are instructions how to use the camera, check that they all work
 
 ### Selection and Scene tree
 
-- Verify that on the left side the panel (scene tree) shows a tree with all of the entities in the loaded model
-- Click on the expand arrow on the left -> The entity shows its child entities
+- Verify the scene tree shows a every model and its children (there may be no children, if the object is converted that way)
```
Comment (MichaelZp0) [inline .pipelines/testchecklist.md:105]: nit - shows every model

**Pull request merged**